buildenv/jenkins/variables/defaults.yml

```diff
@@ -62,6 +66,7 @@ jdk_image_dir:
   8: 'j2sdk-image'
   11: 'jdk'
   12: 'jdk'
+  13: 'jdk'
```

Comment:
Is there a way to make this easier in the future? i.e. we can specify `8:` and a `default:` for everything else? In many cases here and below the values don't change for jdk13.

Reply:
I agree this is painful.
@dnakamura has an old PR #2016 which we should revisit.